src/snowcli/cli/snowpark/compute_pool/commands.py

```diff
@@ -9,8 +8,8 @@
 
 app = typer.Typer(
     context_settings=DEFAULT_CONTEXT_SETTINGS,
-    name="compute-pool",
-    help="Manages compute pools. You can also use `cp` as alias for this command.",
+    name="pool",
+    help="Manage compute pools.",
```

Comment:
Manage -> Manages

I think Mike changed all occurrences without `s` to versions with `s`, so let's stay consistent.

Reply:
```suggestion
    help="Manages compute pools.",
```